devito/mpi/routines.py

```diff
@@ -483,6 +467,27 @@ def _make_haloupdate(self, f, hse, key, sendrecv, **kwargs):
 
         return HaloUpdate('haloupdate%s' % key, iet, parameters)
 
+    def _make_basic_mapper(self, f, fixed):
```

Comment:
Turn the comment at the top into a docstring?

Reply:
done